index.script.js

```diff
@@ -0,0 +1,39 @@
+function add() {
+    let field1=document.getElementById('num1').value ;
+    let field2=document.getElementById('num2').value ;
+
+    let total=parseFloat(field1) + parseFloat(field2) ;
+    if(!isNaN(total))
+    {document.getElementById("answer").innerHTML="The answer is "+total;
+    }
+}
+
+function subtract() {
```

Comment:
also you can combine all the functions into 1 function and use if statements or switch case statements to see which operation to perform. then you will have only one function to maintain.